easybuild/easyconfigs/a/Amber/Amber-22.0-foss-2021b-AmberTools-22.3.eb

```diff
@@ -0,0 +1,96 @@
+name = 'Amber'
+local_amber_ver = 22
+local_ambertools_ver = 22
+# Patch levels from http://ambermd.org/AmberPatches.php and http://ambermd.org/ATPatches.php
+patchlevels = (3, 0)  # (AmberTools, Amber)
+version = '%s.%s' % (local_amber_ver, patchlevels[1])
+versionsuffix = '-AmberTools-%s.%s' % (local_ambertools_ver, patchlevels[0])
+
+homepage = 'http://ambermd.org/amber.html'
```

Comment:
```suggestion
homepage = 'https://ambermd.org/amber.html'
```

Reply:
Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/16166/commits/eef360dc8d0f30d27748cef9452087f11dcbaf4a